heron/tools/tracker/src/python/main.py

```diff
@@ -55,6 +55,8 @@ def __init__(self):
         (r"/topologies/executionstate", handlers.ExecutionStateHandler, {"tracker":self.tracker}),
```

Comment:
I wasn't suggesting removing/refactoring it in this PR, just putting a comment describing that it's slated to be removed. That way if it takes a while to remove people readying the code will understand the duplication.